ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query/Version.hs

```diff
@@ -0,0 +1,8 @@
+
+module Ouroboros.Consensus.Ledger.Query.Version (QueryVersion (..)) where
+
+-- | Version of the `Query blk` type.
+data QueryVersion
+  = TopLevelQueryDisabled
+  | QueryV_1
```

Comment:
Please add some comments to each constructor. (My other comment on this PR reveals my hope that we'll need to update the comment on `QueryV_1` when we subsequently enrich it to actually provide a top-level query.)

Reply:
Good idea!